Frontend/lib/models/tournament.dart

```diff
@@ -0,0 +1,44 @@
+enum StartingMethod {
+  bulloff,
+  random,
+}
+
+class TournamentModel {
+  final String id;
```

Comment:
Ik zou `id` Nullable maken. Omdat als je een TournamentModel maakt maar nog niet opslaat in de database, dan heeft hij nog geen id. De database maakt dan pas een id aan als hij wordt opgeslagen, dan kun je hem ook toevoegen aan TournamentModel.